WebApplicationDemo/Application/TestContext.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Ark.Tools.Outbox;
+using Ark.Tools.Sql;
+
+using Dapper;
+
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using System.Threading;
+using System.Threading.Tasks;
+
+using WebApplicationDemo.Dto;
+
+namespace WebApplicationDemo.Application
+{
+    public class TestContext : AbstractSqlContextAsync<ISqlContextAsync>, ISqlDataContext
```

Comment:
this is not the right way to use. AbstractSqlContextAsync<Tag> should use a Tag class.

check correct usage of the Sync version.